### PR TITLE
CLOUDP-197338: [Atlas CLI] "atlas dbusers certs create" doesn't allow downloading the PEM file

### PR DESCRIPTION
**Pull request by gssbzn**
## Proposed changes

_Jira ticket:_ CLOUDP-197338

## Checklist

Updates the SDK to fix an issue with creating x509 certs, this will probably a breaking change of outputs

### CODE REVIEW

File: internal/store/database_users.go [hunk 1]
```diff
@@ -124,15 +124,15 @@ func (s *Store) DBUserCertificates(projectID, username string, opts *atlas.ListO
...
 		userCert := &atlasv2.UserCert{
 			MonthsUntilExpiration: pointer.Get(monthsUntilExpiration),
 		}
-		_, err := s.clientv2.X509AuthenticationApi.CreateDatabaseUserCertificate(s.ctx, projectID, username, userCert).Execute()
-		return userCert, err
+		cert, _, err := s.clientv2.X509AuthenticationApi.CreateDatabaseUserCertificate(s.ctx, projectID, username, userCert).Execute()
+		return cert, err
 	default:
-		return nil, fmt.Errorf("%w: %s", errUnsupportedService, s.service)
+		return "", fmt.Errorf("%w: %s", errUnsupportedService, s.service)
```
Comment (gssbzn) [inline internal/store/database_users.go:136]: this is the only major change beside lib bump

Comment (wtrocki) [reply internal/store/database_users.go:136]: Is it fine to print the cert or we need file support?

**Review by wtrocki — APPROVED** [review]

File: docs/atlascli/command/atlas-dbusers-certs-create.txt [hunk 2]
```diff
@@ -81,7 +81,7 @@ If the command succeeds, the CLI returns output similar to the following sample.
 
 .. code-block::
 
-   <Certificate>
+   <>
```
Comment (wtrocki) [inline docs/atlascli/command/atlas-dbusers-certs-create.txt:84]: This looks odd in docs.

Comment (gssbzn) [reply docs/atlascli/command/atlas-dbusers-certs-create.txt:84]: agreed

File: internal/store/database_users.go (hunk 1 above)
Comment (gssbzn) [reply internal/store/database_users.go:136]: you mean a `-o` support? feel free to open a ticket as a follow up

**Pull request merged**